site/app/controllers/pdf/PDFController.php

```diff
@@ -66,7 +66,6 @@ public function showStudentPDF($gradeable_id, $filename, $path) {
     /**
      * @param $gradeable_id
      * @Route("/{_semester}/{_course}/gradeable/{gradeable_id}/pdf/annotations", methods={"POST"})
-     * @AccessControl(role="LIMITED_ACCESS_GRADER")
```

Comment:
Since we are removing this, a check should be added below to make sure that the student id acquired by ```$user_id = $annotation_info['user_id'];``` is in our peer grading set.

Reply:
Added a check for peer assignments annotated by students that the student grader is a valid grader for the gradeable and student